rest/route/patch.go

```diff
@@ -646,14 +646,29 @@ func (p *schedulePatchHandler) Run(ctx context.Context) gimlet.Responder {
 		}
 	}
 	patchUpdateReq := dbModel.PatchUpdate{
-		Description: p.variantTasks.Description,
-		Caller:      u.Id,
+		Description:   p.variantTasks.Description,
+		Caller:        u.Id,
+		VariantsTasks: p.patch.VariantsTasks,
```

Comment:
Can we add a comment to `PatchUpdate.VariantsTasks` to clarify that the `PatchUpdate` must include _all_ of the variant tasks for the patch (existing and new ones)? Without reading all the finalization logic, I would have thought it would be sufficient to pass just the diff of variant tasks to be added to the patch.